recon/src/recon.rs

```diff
@@ -279,19 +280,12 @@ where
         self.store.value_for_key(&key).await
     }
 
-    /// Insert key into the key space. Includes an optional value.
-    /// Returns a boolean (true) indicating if the key was new.
-    pub async fn insert(&self, item: &ReconItem<'_, K>) -> Result<bool> {
-        self.store.insert(item).await
-    }
-
-    /// Insert many keys into the key space. Includes an optional value for each key.
-    /// Returns an array with a boolean for each key indicating if the key was new.
-    /// The order is the same as the order of the keys. True means new, false means not new.
-    pub async fn insert_many(&self, items: &[ReconItem<'_, K>]) -> Result<Vec<bool>> {
-        let result = self.store.insert_many(items).await?;
-
-        Ok(result.keys)
+    /// Insert key into the key space.
+    /// Returns Ok if the result was accepted. It may be validated and stored
+    /// out of band, meaning it may not immediately return in range queries.
```

Comment:
is this still true about validation happening out of band?  Didn't you say on our call earlier that this is actually going to be happening inline for now?

Reply:
it's not, but that comment is also changed in #473 😅